docs/spec/index.html

```diff
@@ -1,740 +1,952 @@
+      enrolling in continuous integration tests. These tests are based
+      on Postman collections which orchestrate the interaction of
+      participating parties, such as `Credential Issuance`, `Credential
+      Verification`, and `Presentation Exchange`. These Postman
+      collections are described with a set of
+      <a
+        href="https://github.com/w3c-ccg/traceability-interop/tree/main/docs/tutorials">tutorials</a>
+      that show how to run the tests against a particular implementation.
+    </p>
+    <p>
+      An interoperability report is continuously created from the
+      results of executing these collections, targeting the
+      participating platforms. In cases of multi-party interactions
+      (such as party A making a verifiable presentation to party B), all
+      combinations of platform interchange are executed. The
+      Interoperability report is executed by a GitHub Action on the
+      Traceability Interoperability GitHub repository:
+      <a
+        href="https://github.com/w3c-ccg/traceability-interop/actions/workflows/interoperability-report.yml">interoperability-report.yml</a>
+    </p>
+    <p>
+      In each test, the GitHub Action runner carries out a number of
+      invocations, acting on behalf of varying parties taking different
+      roles (`issuer`, `holder`, and `verifier`), in turn invoking the
+      participating platform.
+    </p>
+    Below is an example of calls made by the runner as it is executing a
+    DIDAuth Presentation Exchange flow:
+    <ul>
+      <li>Runner calls `/credentials/issue` on Party A's platform, representing
+        Party A acting as `issuer`.</li>
+      <li>Runner calls `/presentations/available` on Party B's platform,
+        representing Party A acting as `holder`.</li>
+      <li>Runner calls `/presentations/prove` on Party A's platform,
+        representing Party A acting as `holder`.</li>
+      <li>
+        Runner calls `/presentations/submissions` on Party B's platform,
+        representing Party A acting as `holder`.
+      </li>
+      <li>Runner calls `/credentials/verify` on Party B's platform, representing
+        Party B acting as `verifier`.</li>
+    </ul>
+  </section>
+
+  <section class="informative">
+    <h2>OAS Implementation</h2>
+    <p>
+      The Traceability Interoperability
+      <a href="https://w3id.org/traceability/interoperability/openapi">Open API
+        Specification</a>
+      is designed to be easily implemented and used to access conforming
+      platforms.
+    </p>
+    <p>
+      In order to make use of the provide OpenAPI specifications
+      directly, several placeholder URLs need to be modified to reflect
+      your operating environment:
+    </p>
+    <p>Add the target <code>server</code>:</p>
+    <p></p>
+    <pre class="example">
+          servers:
+            - url: https://conformant-platform.example.com
+   </pre>
+    <p>Add the target <code>tokenUrl</code>:</p>
+    <p></p>
```

Comment:
empty p tag, we might want to add a link to the open AzureAD / scope issue here.